uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -106,16 +144,20 @@ def on_task_complete(self, task):
 
     def handle_connect(self, event):
         self.connect_event = event
-        request = event.h11request
-        headers = [(key.lower(), value) for key, value in request.headers]
-        path, _, query_string = request.target.partition(b"?")
+        # print([x for x in dir(event) if not x.startswith('__')])
+        request = event#.h11request
+        # headers = [(key.lower(), value) for key, value in request.headers]
+        headers = [(key.lower(), value) for key, value in request.extra_headers]
+        #path, _, query_string = request.target.partition(b"?")
+        path, _, query_string = request.target.partition("?")
```

Comment:
`target` and `path` are now a string. I *think* `extra_headers` is what previously was `headers`.

Reply:
Confirmed.